perfkitbenchmarker/linux_benchmarks/netperf_pps_benchmark.py

```diff
@@ -147,43 +147,38 @@ def RunNetperfAggregate(vm, server_ips):
   """
 
   # setup remote hosts file
-  vm.RemoteCommand("cd %s && rm remote_hosts"
-                   % (netperf.NETPERF_EXAMPLE_DIR))
+  vm.RemoteCommand(f"cd {netperf.NETPERF_EXAMPLE_DIR} && rm remote_hosts")
```

Comment:
Use single quotes (it should have been double quotes to start so not your issue)